src/styles/mixins/_frames.scss

```diff
@@ -1,9 +1,35 @@
+@mixin gold-border-shadow {
+    /* prettier-ignore */
+    border: none;
+    margin: 2px;
+    box-shadow:
+        0 0 0 1px var(--tertiary),
+        0 0 0 2px var(--gold-brown),
+        0 0 20px rgba(black, 0.8);
+}
+
 @mixin gold-border {
```

Comment:
This style was only used one place, the trim around the currencies. The new style is 1px thinner and is used in a lot of locations.